do-catalog/src/store/modules/data/catalog.js

```diff
@@ -10,7 +10,7 @@ const catalog = {
     datasetsListCount: 0,
     dataset: {},
     keyVariables: {},
-    variables: {},
+    variables: [],
```

Comment:
Why this change?
Is it tested?

Reply:
Yep, actually, the backend returns an array with the variables.